src/crewai/utilities/events/crew_events.py

```diff
@@ -2,170 +2,97 @@
 
 from pydantic import InstanceOf
 
-from crewai.utilities.events.base_events import CrewEvent
+from crewai.crew import Crew
+from crewai.utilities.events.base_events import BaseEvent
 
 
-class CrewKickoffStartedEvent(CrewEvent):
-    """Event emitted when a crew starts execution"""
-
+class CrewBaseEvent(BaseEvent):
     crew_name: Optional[str]
-    inputs: Optional[Dict[str, Any]]
-    type: str = "crew_kickoff_started"
-    crew: Optional[Any] = None
+    crew: Optional[Crew] = None
 
     def __init__(self, **data):
```

Comment:
I created a base class for crew events, and DRY-ed fingerprinting stuff in this base class.